code/game/objects/items/weapons/storage/briefcase.dm

```diff
@@ -67,6 +67,17 @@
 
 	return
 
+/obj/item/weapon/storage/briefcase/moneybriefcase
+	empty = 1
+
+
+/obj/item/weapon/storage/briefcase/moneybriefcase/New()
+    ..()
+    for(var/slots in 1 to storage_slots)
+        var/obj/item/weapon/spacecash/money = new /obj/item/weapon/spacecash/c1000(src)
+        money.amount = 10
+        contents += money
```

Comment:
Yeah, contents can't contain multiple references on account of being a Byond internal list based on loc.